src/frontend/src/icp/derived/icrc.derived.ts

```diff
@@ -130,3 +131,8 @@ export const enabledIcrcTokens: Readable<IcToken[]> = derived(
 		...$enabledIcrcCustomTokens
 	]
 );
+
+export const enabledIcrcTokensCanisterIds: Readable<LedgerCanisterIdText[]> = derived(
```

Comment:
The name is too generic - it's not clear it's ledger canister ids.

Reply:
changed to enabledIcrcTokensLedgerCanisterIds